cluster/manifests/spot-node-rescheduler/cronjob.yaml

```diff
@@ -0,0 +1,35 @@
+{{ if ne .ConfigItems.spot_node_rescheduler "true" }}
```

Comment:
```suggestion
{{ if eq .ConfigItems.spot_node_rescheduler "true" }}
```

Reply:
Should also have the corresponding change in the deletions.yaml:

Example:

https://github.com/zalando-incubator/kubernetes-on-aws/blob/c5823ca2d00d73c4bb2cc5b365cf80abf4704d42/cluster/manifests/deletions.yaml#L89-L103